client/src/components/EmojiPicker.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+import React from 'react';
+import ReactEmoji from "react-emoji";
+import { filterEmojis } from "../utils/emojis";
+import 'styles/EmojiPicker.scss';
+
+const emojify = (emojis, highlightedIndex) => {
+    const emojiOpts = {
+      emojiType: 'emojione'
+    };
+    let counter = 0;
+    return emojis.map(emoji => {
+            let elementClass = '';
+            const element = emoji.element;// ReactEmoji.emojify(emoji, emojiOpts)[0];
+            if (counter === highlightedIndex)
+                elementClass = 'selected';
+            return (<li key={counter++}
+                        className={elementClass}>
+                        {element}
+                    </li>);
+        }
+    );
+};
+
+const EmojiList = ({
+    emojis,
+    highlightedIndex
+}) => <ul> { emojify(emojis, highlightedIndex) } </ul>
+
```

Comment:
Add a support to render "empty" if there are no elements in the emojis array. This will help with the visuals: now it'll render the `<ul>` regardless which, creates an empty box with no content. By returning a `<span/>` for empty list, we keep the UI clean.

```javascript
const EmojiList = ({emojis, highlightedIndex}) => {
  return emojis.length > 0 ? <ul>{emojify(emojis, highlightedIndex)}</ul> : <span/> // return "empty" if there are no emojis to render
};
```